content/applications/inventory_and_mrp/quality/quality_check_types/picture_check.rst

```diff
@@ -124,33 +125,52 @@ specified in the :guilabel:`Work Order Operation` field on the |QCP| form. If a
-do so, begin by navigating to :menuselection:`Manufacturing --> Operations --> Manufacturing
-Orders`. Select a manufacturing order that includes a work order for which a quality check is
-required. Open the tablet view for that work order by selecting the :guilabel:`Work Orders` tab, and
-then clicking the :guilabel:`📱 (tablet)` button on the order's line.
+*Take a Picture* quality checks configured for work orders **must** be completed from the *Shop
+Floor* module. To do so, begin by navigating to :menuselection:`Manufacturing --> Operations -->
+Manufacturing Orders`. Then, select an |MO| that includes a work order for which a *Take a Picture*
+quality check is required.
+
+On the |MO|, select the :guilabel:`Work Orders` tab, and then click the :guilabel:`Open Work Order
+(square with arrow coming out of it)` button on the line of the work order to be processed. On the
+resulting :guilabel:`Work Orders` pop-up window, click the :guilabel:`Open Shop Floor` button to
+open the *Shop Floor* module.
+
+When accessed from a specific work order, the *Shop Floor* module opens to the page for the work
+center where the order is configured to be processed, and isolates the work order's card, so no
+other cards are shown.
+
+Process the work order's steps until the *Take a Picture* quality check step is reached. Click on
+the step to open a pop-up window that includes instructions for how the picture should be taken.
+After taking the picture, make sure it is stored on the device being used to process the quality
+check (computer, tablet, etc.).
 
-With tablet view open, complete the steps listed on the left side of the screen until the *Take a
-Picture* quality check step is reached. Upon reaching the check, follow the instructions for how to
-take the picture, which appear at the top of the screen.
+Then, click the :guilabel:`Take a Picture` button on the pop-up window to open the device's file
+manager. In the file manager, navigate to the picture, select it, and click :guilabel:`Open` to
+attach it.
 
-After taking the picture, make sure it is stored on the device being used to process the work order
-(computer, tablet, etc.). Then, click the :guilabel:`Take a Picture` button to open the device's
-file manager. In the file manager, navigate to the picture, select it, and click :guilabel:`Open` to
-attach it. Finally, click :guilabel:`Validate` to complete the quality check.
+Finally, click :guilabel:`Validate` at the bottom of the pop-up window to complete the quality
+check. The pop-up window then moves on to the next step of the work order.
 
-.. image:: picture_check/work-order-picture-check.png
+.. image:: picture_check/picture-check-shop-floor.png
    :align: center
-   :alt: A Take a Picture check for a manufacturing work order.
+   :alt: A Take a Picture check in the Shop Floor module.
+
+If a quality alert must be created, exit the pop-up window by clicking the :guilabel:`X` button in
+the top-right corner.
```

Comment:
Whether or not you use an icon here is up to you, but adding the descriptor is necessary to stay consistent.
```suggestion
If a quality alert must be created, exit the pop-up window by clicking the :guilabel:`X (close)`
button in the top-right corner.
```